frontend/src/main/web/lib/constants/Views.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Router from 'react-router';
+import StringUtils from '../utils/StringUtils';
+import RootContent from '../components/RootContent';
+import UserProfile from '../components/UserProfile';
+import SystemGlossary from '../components/SystemGlossary';
+
+var Views = {
+  USER_PROFILE: 0,
+  GLOSSARY: 1,
+
+  getView: function(value) {
+    if(StringUtils.isEmptyOrNull(value)) {
+      return null;
+    }
+
+    switch (value.toLowerCase()) {
+      case 'profile':
+      case 'user_profile':
+      case 'userProfile':
+        return this.USER_PROFILE; break;
+      case 'glossary': return this.GLOSSARY; break;
+      default: return null; break;
```

Comment:
This should only happen if we code something wrong. We want to know about it. Throw an `Error` here.